### PR TITLE
Add xcape page (uses f2py)

### PR DESCRIPTION
**Pull request by rabernat**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

We copied much of this from https://github.com/conda-forge/xlayers-feedstock/blob/master/recipe/meta.yaml, which is quite similar.

### CODE REVIEW

**Comment by rabernat** [comment]
cc @chiaral

**Comment by rabernat** [comment]
@conda-forge/help-python: ready for review.

Note this package uses f2py to create binary modules. I hope we have done it correctly.

File: recipes/xcape/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xgcm/xcape/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<36 or win]
```
Comment (synapticarbors) [inline recipes/xcape/meta.yaml:15]: Do you mean to be skipping `win` here? The `build` section listing a fortran for windows seems contradictory.

Comment (rabernat) [reply recipes/xcape/meta.yaml:15]: We copied this directly from https://github.com/conda-forge/xlayers-feedstock/blob/6fb15babf22e3cf81d2d0531ff86c5fea69499d9/recipe/meta.yaml#L11-L14, which was written by @ocefpaf. TBH, I'm not exactly sure what it does. Should I try removing?

Comment (ocefpaf) [reply recipes/xcape/meta.yaml:15]: We left that in xlayers b/c we do plan to try Windows again later. It is harmless to leave it here too.

**Pull request merged**